components/datatypes/industry-verticals/vehicle-details.schema.json

```diff
@@ -0,0 +1,118 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/industry-verticals/vehicle-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Vehicle Details",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Details specific to a vehicle reservation.",
+  "definitions": {
+    "vehicle-details": {
+      "properties": {
+        "xdm:agentID": {
+          "title": "Agent ID",
+          "type": "string",
+          "description": "Agent ID associated with the vehicle booking."
+        },
+        "xdm:averageDailyPrice": {
+          "title": "Average Daily Price",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/currency",
+          "description": "Average daily price of the vehicle."
+        },
+        "xdm:basePrice": {
+          "title": "Base Price",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/currency",
+          "description": "Base price before any discounts are added. "
+        },
+        "xdm:couponCode": {
+          "title": "Coupon Code",
+          "type": "string",
+          "description": "Coupon code associated with the vehicle booking."
+        },
+        "xdm:currencyCode": {
+          "title": "Currency Code",
+          "type": "string",
+          "description": "Currency code (i.e. 'USD' or 'GBP') used to make the purchase."
```

Comment:
Can you please add the  "pattern": "^[A-Z]{3}$" ?
Also it will be informative to add  "ISO 4217 currency code." to the description